notebooks/neo4j_to_networkx.ipynb

```diff
@@ -0,0 +1,434 @@
+{
```

Comment:
Just to note the page paths throughout which direct users to read or save a csv can be changed to, e.g.: 
```
nx.write_gpickle(g6, "..data/processed/structural_graph.gpickle")
```
and for example
```
page_paths = pd.read_csv("..data/processed/page_paths.csv")
```